memory/ndctl_selftest.py

```diff
@@ -86,16 +86,6 @@ def setUp(self):
 
         self.sourcedir = self.teststmpdir
         os.chdir(self.sourcedir)
-
-        # TODO: remove patches once merged upstream
-        upstream_patch = self.fetch_asset("upstream.patch", locations=[
-            "https://patchwork.kernel.org/series/177255/mbox/"], expire='7d')
-
-        process.run('patch -p1 < %s' % upstream_patch, shell=True)
-        if detected_distro.arch in ["ppc64", "ppc64le"]:
-            process.run('patch -p1 < %s' %
-                        self.get_data('ppc.patch'), shell=True)
-
```

Comment:
@SamirMulani are we removing the upstream support here?

Reply:
no, it like patch already part for ndctl upstream

Reply:
Tried and tested the code, removing the above block of code snippet. 
Changes look good & we can safely go ahead for the merge.